ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -188,13 +188,45 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
         config.getConfigParameter<double>(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
 
+    // Non-equilibrium variables
+    ParameterLib::Parameter<double> const* nonequilibrium_stress = nullptr;
```

Comment:
Just out of curiosity: Why not ´unique_ptr`?

Reply:
It's a non-owning optional parameter.